px-alert-message.html

```diff
@@ -59,22 +59,22 @@
       </template>
       <div class="message-column flex flex--middle">
         <div class="message-container">
-          <div class$="{{type}} message collapsed-message" id="message">
+          <div class$="[[_getMessageClassNames(type, expanded)]]" id="message">
             <span class="title">{{ messageTitle }}</span>
             <span>{{ message }}</span>
           </div>
-          <span id="showMoreButton" class="actionable visuallyhidden show-more">[[localize(_showBtnText)]]</span>
+          <span id="showMoreButton" class="actionable show-more" hidden$="[[!overset]]">[[_showBtnText]]</span>
```

Comment:
Hmm....
1) localize should be maintained I assume? Or is this an obsolete localization and not Ben's approach?
2) I think we want to continue using visuallyhidden in this case for ally